### ISSUE TITLE
Crash slicing

### ISSUE THREAD

**Issue by superfloh247** [SoftFever/OrcaSlicer#3178]
### OrcaSlicer Version

0ab5601

### OS version

MacOS 14.2

### Additional system information

MacOS 14.2, Xcode 15.1, AppleSilicon M1pro

### Printer

Bambu P1P

### How to reproduce

Slice file, see attachments

stl source https://www.thingiverse.com/thing:5027648

### Actual results

crash

```
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               OrcaSlicer [12205]
Path:                  /Users/USER/*/OrcaSlicer.app/Contents/MacOS/OrcaSlicer
Identifier:            com.softfever3d.orca-slicer
Version:               1.9.0-dev (???)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               502

Date/Time:             2023-12-18 15:03:02.0242 +0100
OS Version:            macOS 14.2 (23C64)
Report Version:        12
Anonymous UUID:        D2A190B5-E540-B691-B8D3-A8F39A52196D


Time Awake Since Boot: 6600 seconds

System Integrity Protection: enabled

Crashed Thread:        33  bbl_BgSlcPcs

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000970
Exception Codes:       0x0000000000000001, 0x0000000000000970

Termination Reason:    Namespace SIGNAL, Code 11 Segmentation fault: 11
Terminating Process:   exc handler [12205]

VM Region Info: 0x970 is not in any region.  Bytes before following region: 4295751312
      REGION TYPE                    START - END         [ VSIZE] PRT/MAX SHRMOD  REGION DETAIL
      UNUSED SPACE AT START
--->  
      __TEXT                      1000c0000-103c68000    [ 59.7M] r-x/r-x SM=COW  ...OS/OrcaSlicer

Thread 0:: orcaslicer_main Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	       0x18c2b1874 mach_msg2_trap + 8
1   libsystem_kernel.dylib        	       0x18c2c3cf0 mach_msg2_internal + 80
2   libsystem_kernel.dylib        	       0x18c2ba4b0 mach_msg_overwrite + 476
3   libsystem_kernel.dylib        	       0x18c2b1bf8 mach_msg + 24
4   CoreFoundation                	       0x18c3cfbf4 __CFRunLoopServiceMachPort + 160
5   CoreFoundation                	       0x18c3ce4bc __CFRunLoopRun + 1208
6   CoreFoundation                	       0x18c3cd9ac CFRunLoopRunSpecific + 608
7   HIToolbox                     	       0x19697c448 RunCurrentEventLoopInMode + 292
8   HIToolbox                     	       0x19697c284 ReceiveNextEventCommon + 648
9   HIToolbox                     	       0x19697bfdc _BlockUntilNextEventMatchingListInModeWithFilter + 76
10  AppKit                        	       0x18fbaa8a4 _DPSNextEvent + 660
11  AppKit                        	       0x190384980 -[NSApplication(NSEventRouting) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 716
12  AppKit                        	       0x18fb9dd50 -[NSApplication run] + 476
13  OrcaSlicer                    	       0x102e55458 wxGUIEventLoop::OSXDoRun() + 140
14  OrcaSlicer                    	       0x102f1d240 wxCFEventLoop::DoRun() + 40
15  OrcaSlicer                    	       0x102ea7844 wxEventLoopBase::Run() + 120
16  OrcaSlicer                    	       0x102e85288 wxAppConsoleBase::MainLoop() + 120
17  OrcaSlicer                    	       0x102e1a2bc wxApp::OnRun() + 36
18  OrcaSlicer                    	       0x102ecf170 wxEntry(int&, wchar_t**) + 88
19  OrcaSlicer                    	       0x100eeb4c0 Slic3r::GUI::GUI_Run(Slic3r::GUI::GUI_InitParams&) + 184
20  OrcaSlicer                    	       0x1000d2ccc Slic3r::CLI::run(int, char**) + 29256
21  OrcaSlicer                    	       0x10011b4d0 main + 172
22  dyld                          	       0x18bf710e0 start + 2360

Thread 1:
0   libsystem_pthread.dylib       	       0x18c2ece28 start_wqthread + 0

Thread 2:
0   libsystem_pthread.dylib       	       0x18c2ece28 start_wqthread + 0

Thread 3:: com.apple.NSEventThread
0   libsystem_kernel.dylib        	       0x18c2b1874 mach_msg2_trap + 8
1   libsystem_kernel.dylib        	       0x18c2c3cf0 mach_msg2_internal + 80
2   libsystem_kernel.dylib        	       0x18c2ba4b0 mach_msg_overwrite + 476
3   libsystem_kernel.dylib        	       0x18c2b1bf8 mach_msg + 24
4   CoreFoundation                	       0x18c3cfbf4 __CFRunLoopServiceMachPort + 160
5   CoreFoundation                	       0x18c3ce4bc __CFRunLoopRun + 1208
6   CoreFoundation                	       0x18c3cd9ac CFRunLoopRunSpecific + 608
7   AppKit                        	       0x18fcd4510 _NSEventThread + 144
8   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
9   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 4:: iotc_conn_hdlr
0   libsystem_kernel.dylib        	       0x18c2bc9d4 __select + 8
1   libBambuSource.dylib          	       0x10c93d9dc tutk_platform_fdset_select + 112
2   libBambuSource.dylib          	       0x10c93f440 connectionHandlerThread + 404
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 5:: iotc_taskmgr
0   libsystem_kernel.dylib        	       0x18c2b4e68 __semwait_signal + 8
1   libsystem_c.dylib             	       0x18c1942f8 nanosleep + 220
2   libsystem_c.dylib             	       0x18c194210 usleep + 68
3   libBambuSource.dylib          	       0x10c93e1dc tutk_TaskMng_Run + 396
4   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
5   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 6:
0   libsystem_kernel.dylib        	       0x18c2b506c __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x18c2f25fc _pthread_cond_wait + 1228
2   libc++.1.dylib                	       0x18c219570 std::__1::condition_variable::__do_timed_wait(std::__1::unique_lock<std::__1::mutex>&, std::__1::chrono::time_point<std::__1::chrono::system_clock, std::__1::chrono::duration<long long, std::__1::ratio<1l, 1000000000l>>>) + 100
3   libbambu_networking.dylib     	       0x10b052dd8 BBL::TunnelMqtt::run() + 956
4   libbambu_networking.dylib     	       0x10b0560f8 void* std::__1::__thread_proxy<std::__1::tuple<std::__1::unique_ptr<std::__1::__thread_struct, std::__1::default_delete<std::__1::__thread_struct>>, BBL::TunnelMqtt::attached()::$_1>>(void*) + 68
5   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
6   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 7:: JavaScriptCore libpas scavenger
0   libsystem_kernel.dylib        	       0x18c2b506c __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x18c2f25fc _pthread_cond_wait + 1228
2   JavaScriptCore                	       0x1a95c9550 scavenger_thread_main + 1512
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 8:: com.apple.coreanimation.render-server
0   libsystem_kernel.dylib        	       0x18c2b1874 mach_msg2_trap + 8
1   libsystem_kernel.dylib        	       0x18c2c3cf0 mach_msg2_internal + 80
2   libsystem_kernel.dylib        	       0x18c2ba4b0 mach_msg_overwrite + 476
3   libsystem_kernel.dylib        	       0x18c2b1bf8 mach_msg + 24
4   QuartzCore                    	       0x1944d5fa4 CA::Render::Server::server_thread(void*) + 544
5   QuartzCore                    	       0x1944d5d74 thread_fun(void*) + 32
6   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
7   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 9:
0   libsystem_pthread.dylib       	       0x18c2ece28 start_wqthread + 0

Thread 10:: WebCore: Scrolling
0   libsystem_kernel.dylib        	       0x18c2b1874 mach_msg2_trap + 8
1   libsystem_kernel.dylib        	       0x18c2c3cf0 mach_msg2_internal + 80
2   libsystem_kernel.dylib        	       0x18c2ba4b0 mach_msg_overwrite + 476
3   libsystem_kernel.dylib        	       0x18c2b1bf8 mach_msg + 24
4   CoreFoundation                	       0x18c3cfbf4 __CFRunLoopServiceMachPort + 160
5   CoreFoundation                	       0x18c3ce4bc __CFRunLoopRun + 1208
6   CoreFoundation                	       0x18c3cd9ac CFRunLoopRunSpecific + 608
7   CoreFoundation                	       0x18c44b704 CFRunLoopRun + 64
8   JavaScriptCore                	       0x1a81e24a4 WTF::Detail::CallableWrapper<WTF::RunLoop::create(char const*, WTF::ThreadType, WTF::Thread::QOS)::$_1, void>::call() + 52
9   JavaScriptCore                	       0x1a81fe830 WTF::Thread::entryPoint(WTF::Thread::NewThreadContext*) + 244
10  JavaScriptCore                	       0x1a8059ae4 WTF::wtfThreadEntryPoint(void*) + 16
11  libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
12  libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 11:
0   libsystem_kernel.dylib        	       0x18c2b506c __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x18c2f25fc _pthread_cond_wait + 1228
2   libc++.1.dylib                	       0x18c2194dc std::__1::condition_variable::wait(std::__1::unique_lock<std::__1::mutex>&) + 28
3   OrcaSlicer                    	       0x10105bcfc Slic3r::GUI::BoostThreadWorker::run() + 164
4   OrcaSlicer                    	       0x1015d48e0 boost::(anonymous namespace)::thread_proxy(void*) + 188
5   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
6   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 12:
0   libsystem_kernel.dylib        	       0x18c2b506c __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x18c2f25fc _pthread_cond_wait + 1228
2   OrcaSlicer                    	       0x1010d2524 Slic3r::GUI::MediaPlayCtrl::media_proc() + 352
3   OrcaSlicer                    	       0x1015d48e0 boost::(anonymous namespace)::thread_proxy(void*) + 188
4   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
5   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 13:: slic3r_tbb_5
0   libsystem_kernel.dylib        	       0x18c2b17f0 semaphore_wait_trap + 8
1   OrcaSlicer                    	       0x102b15d9c tbb::detail::r1::rml::private_worker::run() + 412
2   OrcaSlicer                    	       0x102b15bf4 tbb::detail::r1::rml::private_worker::thread_routine(void*) + 12
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 14:: slic3r_tbb_6
0   libsystem_kernel.dylib        	       0x18c2b17f0 semaphore_wait_trap + 8
1   OrcaSlicer                    	       0x102b15d9c tbb::detail::r1::rml::private_worker::run() + 412
2   OrcaSlicer                    	       0x102b15bf4 tbb::detail::r1::rml::private_worker::thread_routine(void*) + 12
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 15:: slic3r_tbb_1
0   libsystem_kernel.dylib        	       0x18c2b17f0 semaphore_wait_trap + 8
1   OrcaSlicer                    	       0x102b15d9c tbb::detail::r1::rml::private_worker::run() + 412
2   OrcaSlicer                    	       0x102b15bf4 tbb::detail::r1::rml::private_worker::thread_routine(void*) + 12
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 16:: slic3r_tbb_7
0   libsystem_kernel.dylib        	       0x18c2b17f0 semaphore_wait_trap + 8
1   OrcaSlicer                    	       0x102b15d9c tbb::detail::r1::rml::private_worker::run() + 412
2   OrcaSlicer                    	       0x102b15bf4 tbb::detail::r1::rml::private_worker::thread_routine(void*) + 12
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 17:: slic3r_tbb_3
0   libsystem_kernel.dylib        	       0x18c2b17f0 semaphore_wait_trap + 8
1   OrcaSlicer                    	       0x102b15d9c tbb::detail::r1::rml::private_worker::run() + 412
2   OrcaSlicer                    	       0x102b15bf4 tbb::detail::r1::rml::private_worker::thread_routine(void*) + 12
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 18:: slic3r_tbb_4
0   libsystem_kernel.dylib        	       0x18c2b17f0 semaphore_wait_trap + 8
1   OrcaSlicer                    	       0x102b15d9c tbb::detail::r1::rml::private_worker::run() + 412
2   OrcaSlicer                    	       0x102b15bf4 tbb::detail::r1::rml::private_worker::thread_routine(void*) + 12
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 19:: slic3r_tbb_8
0   libsystem_kernel.dylib        	       0x18c2b17f0 semaphore_wait_trap + 8
1   OrcaSlicer                    	       0x102b15d9c tbb::detail::r1::rml::private_worker::run() + 412
2   OrcaSlicer                    	       0x102b15bf4 tbb::detail::r1::rml::private_worker::thread_routine(void*) + 12
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 20:: slic3r_tbb_2
0   libsystem_kernel.dylib        	       0x18c2b17f0 semaphore_wait_trap + 8
1   OrcaSlicer                    	       0x102b15d9c tbb::detail::r1::rml::private_worker::run() + 412
2   OrcaSlicer                    	       0x102b15bf4 tbb::detail::r1::rml::private_worker::thread_routine(void*) + 12
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 21:: slic3r_tbb_9
0   libsystem_kernel.dylib        	       0x18c2b17f0 semaphore_wait_trap + 8
1   OrcaSlicer                    	       0x102b15d9c tbb::detail::r1::rml::private_worker::run() + 412
2   OrcaSlicer                    	       0x102b15bf4 tbb::detail::r1::rml::private_worker::thread_routine(void*) + 12
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 22:
0   libsystem_kernel.dylib        	       0x18c2b506c __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x18c2f25fc _pthread_cond_wait + 1228
2   OrcaSlicer                    	       0x1015d5384 boost::condition_variable::do_wait_until(boost::unique_lock<boost::mutex>&, boost::detail::real_platform_timepoint const&) + 84
3   OrcaSlicer                    	       0x100431a80 Slic3r::_BBS_Backup_Manager::operator()() + 572
4   OrcaSlicer                    	       0x1015d48e0 boost::(anonymous namespace)::thread_proxy(void*) + 188
5   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
6   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 23:
0   libsystem_pthread.dylib       	       0x18c2ece28 start_wqthread + 0

Thread 24:
0   libsystem_pthread.dylib       	       0x18c2ece28 start_wqthread + 0

Thread 25:
0   libsystem_kernel.dylib        	       0x18c2b506c __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x18c2f25fc _pthread_cond_wait + 1228
2   OrcaSlicer                    	       0x1015d5384 boost::condition_variable::do_wait_until(boost::unique_lock<boost::mutex>&, boost::detail::real_platform_timepoint const&) + 84
3   OrcaSlicer                    	       0x100db3c70 void boost::this_thread::sleep_for<long long, boost::ratio<1l, 1000l>>(boost::chrono::duration<long long, boost::ratio<1l, 1000l>> const&) + 220
4   OrcaSlicer                    	       0x100ec1dcc boost::detail::thread_data<Slic3r::GUI::GUI_App::start_sync_user_preset(bool)::$_1>::run() + 684
5   OrcaSlicer                    	       0x1015d48e0 boost::(anonymous namespace)::thread_proxy(void*) + 188
6   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
7   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 26:
0   libsystem_kernel.dylib        	       0x18c2bc9d4 __select + 8
1   libbambu_networking.dylib     	       0x10afa1cd4 BBL::SsdpDiscovery::ssdp_thread() + 200
2   libbambu_networking.dylib     	       0x10b2dc4dc boost::(anonymous namespace)::thread_proxy(void*) + 176
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 27:
0   libsystem_kernel.dylib        	       0x18c2b506c __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x18c2f25fc _pthread_cond_wait + 1228
2   libbambu_networking.dylib     	       0x10b07e058 Thread_wait_cond + 104
3   libbambu_networking.dylib     	       0x10b06dc28 MQTTAsync_sendThread + 3632
4   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
5   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 28:
0   libsystem_kernel.dylib        	       0x18c2bc9d4 __select + 8
1   libbambu_networking.dylib     	       0x10b079abc Socket_getReadySocket + 352
2   libbambu_networking.dylib     	       0x10b06e6a8 MQTTAsync_receiveThread + 504
3   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
4   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 29:
0   libsystem_kernel.dylib        	       0x18c2b506c __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x18c2f25fc _pthread_cond_wait + 1228
2   libc++.1.dylib                	       0x18c2194dc std::__1::condition_variable::wait(std::__1::unique_lock<std::__1::mutex>&) + 28
3   libBambuSource.dylib          	       0x10c8dbd44 CloseTaskQueue::run() + 136
4   libBambuSource.dylib          	       0x10c8dbc5c void* std::__1::__thread_proxy<std::__1::tuple<std::__1::unique_ptr<std::__1::__thread_struct, std::__1::default_delete<std::__1::__thread_struct>>, CloseTaskQueue::CloseTaskQueue()::'lambda'()>>(void*) + 44
5   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
6   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 30:: avConnReadThread
0   libsystem_kernel.dylib        	       0x18c2b4e68 __semwait_signal + 8
1   libsystem_c.dylib             	       0x18c1942f8 nanosleep + 220
2   libsystem_c.dylib             	       0x18c194210 usleep + 68
3   libBambuSource.dylib          	       0x10c922a98 IOTC_Session_Has_Data + 240
4   libBambuSource.dylib          	       0x10c947de8 IOTC_sCHL_read + 64
5   libBambuSource.dylib          	       0x10c905928 avConnReadThread + 500
6   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
7   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 31:: threadBBR[0][0]
0   libsystem_kernel.dylib        	       0x18c2b4e68 __semwait_signal + 8
1   libsystem_c.dylib             	       0x18c1942f8 nanosleep + 220
2   libsystem_c.dylib             	       0x18c194210 usleep + 68
3   libBambuSource.dylib          	       0x10c8ed160 threadBBR + 376
4   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
5   libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8

Thread 32:
0   libsystem_pthread.dylib       	       0x18c2ece28 start_wqthread + 0

Thread 33 Crashed:: bbl_BgSlcPcs
0   OrcaSlicer                    	       0x1004754f4 Slic3r::GCode::process_layers(Slic3r::Print const&, Slic3r::ToolOrdering const&, std::__1::vector<Slic3r::PrintInstance const*, std::__1::allocator<Slic3r::PrintInstance const*>> const&, std::__1::vector<std::__1::pair<double, std::__1::vector<Slic3r::GCode::LayerToPrint, std::__1::allocator<Slic3r::GCode::LayerToPrint>>>, std::__1::allocator<std::__1::pair<double, std::__1::vector<Slic3r::GCode::LayerToPrint, std::__1::allocator<Slic3r::GCode::LayerToPrint>>>>> const&, Slic3r::GCode::GCodeOutputStream&) + 320
1   OrcaSlicer                    	       0x10046eedc Slic3r::GCode::_do_export(Slic3r::Print&, Slic3r::GCode::GCodeOutputStream&, std::__1::function<std::__1::vector<Slic3r::ThumbnailData, std::__1::allocator<Slic3r::ThumbnailData>> (Slic3r::ThumbnailsParams const&)>) + 30732
2   OrcaSlicer                    	       0x10046582c Slic3r::GCode::do_export(Slic3r::Print*, char const*, Slic3r::GCodeProcessorResult*, std::__1::function<std::__1::vector<Slic3r::ThumbnailData, std::__1::allocator<Slic3r::ThumbnailData>> (Slic3r::ThumbnailsParams const&)>) + 4148
3   OrcaSlicer                    	       0x1008e39e8 Slic3r::Print::export_gcode(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, Slic3r::GCodeProcessorResult*, std::__1::function<std::__1::vector<Slic3r::ThumbnailData, std::__1::allocator<Slic3r::ThumbnailData>> (Slic3r::ThumbnailsParams const&)>) + 568
4   OrcaSlicer                    	       0x100c5c63c Slic3r::BackgroundSlicingProcess::process_fff() + 2060
5   OrcaSlicer                    	       0x100c5fe80 Slic3r::BackgroundSlicingProcess::call_process(std::exception_ptr&) + 64
6   OrcaSlicer                    	       0x100c5ede0 Slic3r::BackgroundSlicingProcess::thread_proc() + 340
7   OrcaSlicer                    	       0x100c627c0 boost::detail::thread_data<Slic3r::BackgroundSlicingProcess::start()::$_3>::run() + 16
8   OrcaSlicer                    	       0x1015d48e0 boost::(anonymous namespace)::thread_proxy(void*) + 188
9   libsystem_pthread.dylib       	       0x18c2f2034 _pthread_start + 136
10  libsystem_pthread.dylib       	       0x18c2ece3c thread_start + 8


Thread 33 crashed with ARM Thread State (64-bit):
    x0: 0x000060000184a050   x1: 0x0000000000000000   x2: 0x0000000000000000   x3: 0x000060000184a060
    x4: 0x000060000184a04c   x5: 0x000060000138146c   x6: 0x000060000184a040   x7: 0x000060000322c000
    x8: 0x0000000000000200   x9: 0x00000000000001ff  x10: 0x0000000000002040  x11: 0x00000000aae1a0c2
   x12: 0x00000000000007fb  x13: 0x00000000000007fd  x14: 0x00000000ab01a903  x15: 0x0000000000000103
   x16: 0x00000000aae1a0c2  x17: 0x000000000001a800  x18: 0x0000000000000000  x19: 0x0000000176599740
   x20: 0x00000001765994c0  x21: 0x0000000176598870  x22: 0x0000000176599850  x23: 0x0000000000000000
   x24: 0x0000000138144200  x25: 0x0000000000000000  x26: 0x0000000000000001  x27: 0x0000000000000000
   x28: 0x0000000000000003   fp: 0x0000000176598490   lr: 0xbb340001004754f0
    sp: 0x0000000176598350   pc: 0x00000001004754f4 cpsr: 0x60001000
   far: 0x0000000000000970  esr: 0x92000046 (Data Abort) byte write Translation fault

Binary Images:
       0x115200000 -        0x115adbfff com.apple.audio.codecs.Components (7.0) <8479fc6b-3097-3c2f-b9ac-e1ece564d865> /System/Library/Components/AudioCodecs.component/Contents/MacOS/AudioCodecs
       0x10c3a0000 -        0x10c40bfff com.apple.AppleMetalOpenGLRenderer (1.0) <41cb4d99-6a07-366a-9e97-d047ce10daa2> /System/Library/Extensions/AppleMetalOpenGLRenderer.bundle/Contents/MacOS/AppleMetalOpenGLRenderer
       0x10c8c8000 -        0x10cba7fff libBambuSource.dylib (*) <ebea71b9-96b6-31ae-85a7-0aa3a85da5d9> /Users/USER/Library/Application Support/OrcaSlicer/*/libBambuSource.dylib
       0x10af50000 -        0x10b393fff libbambu_networking.dylib (*) <cabc0798-5985-31c7-a2c2-fe377499c998> /Users/USER/Library/Application Support/OrcaSlicer/*/libbambu_networking.dylib
       0x10a1f4000 -        0x10a1fffff libobjc-trampolines.dylib (*) <7778e0d7-361a-378d-9438-3b2bb48c2154> /usr/lib/libobjc-trampolines.dylib
       0x1000c0000 -        0x103c67fff com.softfever3d.orca-slicer (1.9.0-dev) <74801ea7-6377-3e15-9a3a-a6196a7d0b8f> /Users/USER/*/OrcaSlicer.app/Contents/MacOS/OrcaSlicer
       0x18c2b0000 -        0x18c2eafff libsystem_kernel.dylib (*) <ca94fc21-bc40-3b43-b65d-b87ece9e1d48> /usr/lib/system/libsystem_kernel.dylib
       0x18c352000 -        0x18c829fff com.apple.CoreFoundation (6.9) <47e4ec09-8f6e-30a8-99d0-34024d4f8122> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
       0x19694c000 -        0x196c0bfff com.apple.HIToolbox (2.1.1) <e2187dfe-1fb4-3b47-874e-a0a6291e51b2> /System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/HIToolbox.framework/Versions/A/HIToolbox
       0x18fb71000 -        0x190e8bfff com.apple.AppKit (6.9) <f3527312-e426-3f7c-b77b-2bf49d1b7c04> /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
       0x18bf6b000 -        0x18bfff347 dyld (*) <324e4ad9-e01f-3183-b09f-3e20b326643a> /usr/lib/dyld
               0x0 - 0xffffffffffffffff ??? (*) <00000000-0000-0000-0000-000000000000> ???
       0x18c2eb000 -        0x18c2f7ff3 libsystem_pthread.dylib (*) <a7d94c96-7b1f-3229-9bea-048d037c3292> /usr/lib/system/libsystem_pthread.dylib
       0x18c187000 -        0x18c205ffb libsystem_c.dylib (*) <ab8a0a0b-9a1e-3284-9243-5a0334468ae5> /usr/lib/system/libsystem_c.dylib
       0x18c206000 -        0x18c293ff7 libc++.1.dylib (*) <375254eb-5a65-35d0-b6b4-773e6763d289> /usr/lib/libc++.1.dylib
       0x1a8050000 -        0x1a9727fbf com.apple.JavaScriptCore (19617) <11f9a40a-f6ac-36bb-a3f2-cd03458de1ba> /System/Library/Frameworks/JavaScriptCore.framework/Versions/A/JavaScriptCore
       0x19448e000 -        0x194856fff com.apple.QuartzCore (1.11) <4a615554-86bd-36e4-aab6-39c377bdae2c> /System/Library/Frameworks/QuartzCore.framework/Versions/A/QuartzCore

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0

VM Region Summary:
ReadOnly portion of Libraries: Total=1.7G resident=0K(0%) swapped_out_or_unallocated=1.7G(100%)
Writable regions: Total=6.3G written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=6.3G(100%)

                                VIRTUAL   REGION 
REGION TYPE                        SIZE    COUNT (non-coalesced) 
===========                     =======  ======= 
Accelerate framework               256K        2 
Activity Tracing                   256K        1 
CG image                          17.7M       88 
CG raster data                    13.0M       21 
ColorSync                          720K       37 
CoreAnimation                     4736K       63 
CoreGraphics                        48K        3 
CoreImage                           16K        1 
CoreUI image data                 2544K       18 
Foundation                          16K        1 
Kernel Alloc Once                   32K        1 
MALLOC                             2.1G       85 
MALLOC guard page                  384K       24 
OpenGL GLSL                        256K        3 
STACK GUARD                       56.5M       34 
Stack                             68.4M       47 
VM_ALLOCATE                      132.7M      195 
VM_ALLOCATE (reserved)             3.9G        1         reserved VM address space (unallocated)
WebKit Malloc                    160.1M        3 
WebKit Malloc (reserved)          32.0M        1         reserved VM address space (unallocated)
__AUTH                            3081K      439 
__AUTH_CONST                      33.8M      662 
__CTF                               824        1 
__DATA                            19.3M      652 
__DATA_CONST                      36.4M      670 
__DATA_DIRTY                      1995K      243 
__FONT_DATA                          4K        1 
__GLSLBUILTINS                    5174K        1 
__LINKEDIT                       925.9M        7 
__OBJC_RO                         71.1M        1 
__OBJC_RW                         2168K        1 
__TEXT                           840.2M      691 
dyld private memory                320K        4 
mapped file                      289.7M       46 
shared memory                     1808K       74 
===========                     =======  ======= 
TOTAL                              8.6G     4122 
TOTAL, minus reserved VM space     4.7G     4122 

(truncated)
```

### Expected results

no crash

### Project file & Debug log uploads

[McLaren_Logo.3mf.zip](https://github.com/SoftFever/OrcaSlicer/files/13704733/McLaren_Logo.3mf.zip)

[logs.zip](https://github.com/SoftFever/OrcaSlicer/files/13704742/logs.zip)

### Checklist of files to include

- [x] Log file
- [x] Project file

**Comment by superfloh247** [comment]
OrcaSlicer 1.8.1 release version: not affected

**Comment by superfloh247** [comment]
fixed in 1a1760f
